wagtail/wagtailadmin/static/wagtailadmin/css/.gitignore

```diff
@@ -0,0 +1,9 @@
+core.css
+layouts/home.css
+layouts/login.css
+layouts/page-editor.css
+layouts/preview.css
+normalize.css
+panels/rich-text.css
+userbar.css
+userbar_embed.css
```

Comment:
Should we put all the generated code into a subdirectory so that we don't have to manually set the `.gitignore`?

Something like `wagtailadmin/css/dist/` or similar?